src/components/tutorial-collection-cards/components/card-eyebrow/index.tsx

```diff
@@ -1,12 +1,15 @@
+import classNames from 'classnames'
+import { ReactElement } from 'react'
 import s from './card-eyebrow.module.css'
 
-function CardEyebrow({ text, icon }: { text: string; icon?: JSX.Element }) {
-	return (
-		<span className={s.eyebrow}>
-			{icon ? icon : null}
-			{text}
-		</span>
-	)
+function CardEyebrow({
+	children,
+	className,
+}: {
+	children: ReactElement[] | ReactElement
+	className?: string
+}) {
+	return <span className={classNames(s.eyebrow, className)}>{children}</span>
```

Comment:
I refactored this to pass children. I think this approach will be helpful, especially as we start to have many eyebrow states with progress etc.